From 2035e846bcb75c8579cf7369ab6ba3411a72142c Mon Sep 17 00:00:00 2001
From: antznin <antonin.godard@telecom-paristech.fr>
Date: Mon, 20 Apr 2020 10:25:12 +0000
Subject: [PATCH] Set correct permissions to notify socket

---
 notify_socket.go | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/notify_socket.go b/notify_socket.go
index 316196e..178b4a7 100644
--- a/src/import/notify_socket.go
+++ b/src/import/notify_socket.go
@@ -6,6 +6,7 @@ import (
 	"bytes"
 	"fmt"
 	"net"
+    "os"
 	"path/filepath"
 
 	"github.com/opencontainers/runtime-spec/specs-go"
@@ -60,6 +61,12 @@ func (s *notifySocket) setupSocket() error {
 		return err
 	}
 
+ 	err = os.Chmod(s.socketPath, 0777)
+	if err != nil {
+		socket.Close()
+		return err
+	}
+
 	s.socket = socket
 	return nil
 }
-- 
2.7.4

